display_testing/pil_test2.py

```diff
@@ -0,0 +1,21 @@
+from tkinter import * # version 3.x
```

Comment:
Short header comments describing what these extra files do would be good, even though this is just a testing script.